infrastructure/zk/src/config.ts

```diff
@@ -112,6 +172,17 @@ export function compileConfig(environment?: string) {
     console.log(`Configs compiled for ${environment}`);
 }
 
+// If running ValidiumMode config files should be updated and recompiled to get the right constant values
+// UpdateConfig should be run on init
+export function updateConfig(validiumMode: boolean) {
+    updateChainConfig(validiumMode);
+    updateEthSenderConfig(validiumMode);
```

Comment:
I guess this no longer will be needed after https://github.com/matter-labs/zksync-era/pull/1238 is merged